script/setup/install-critools

```diff
@@ -18,12 +18,12 @@
 # Builds and installs critools including critest and crictl
 # to /usr/local/bin.
 #
-set -eu -o pipefail
```

Comment:
no idea why remove this option

Reply:
`man bash` will tell you:

_The return status of a pipeline is the exit status of the last command, unless the pipefail option is enabled.  If  pipefail  is  enabled,  the pipeline's  return  status is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands  exit  successfully.  If the reserved word !  precedes a pipeline, the exit status of that pipeline is the logical negation of the exit status  as  described above.   The  shell waits for all commands in the pipeline to terminate before returning a value._

For the scripts here that do not have pipes, using the bash-specific -o pipefail does no difference since there are no pipes. Since those scripts contain no further bash-specifics, there is no reason for them to use anything but the default shell (which, of course, could be bash, but as written is not by default in e.g. Debian, Ubuntu, etc.).

For the scripts here that do have pipes, I have done no changes since their current behavior depends on having pipefail.

Reply:
I would like to keep `-o pipefail` in shell script. IMO, shell script should run with `set -euo pipefail` by default. It can save time when meet error. If someone adds scripts with pipes, they have to remember to add it back. So why not just keep it here?

Reply:
They would have to add it back if they wanted the behavior that pipefail gives for the code they would add, yes. But I will re-add the unused pipefail as requested.